tests/check_daemon.c

```diff
@@ -298,7 +298,8 @@ START_TEST(check_process_record_with_correct_size_and_data)
                         "system_name: clear-linux-os\n"
                         "board_name: Qemu|Intel\n"
                         "cpu_model: Intel(R) Core(TM) i7-5650U CPU @ 2.20GHz\n"
-                        "bios_version: Qemu\n";
+                        "bios_version: Qemu\n"
+                        "event_id: 489d80c2f2aba1ff3c7530d0\n";
```

Comment:
The test say "check_process_record_with_correct_size_and_data", but this has an invalidly sized event_id.